### PR TITLE
Basic mvp flow / Only structure

### PR DESCRIPTION
**Pull request by Matias-Villalobo**
The mvp model has been implemented. It should be clarified that the functions are not yet implemented, only the project structure was created.

image

### CODE REVIEW

File: app/src/main/java/com/example/myweatherapp/mvp/presenter/MyWeatherAppPresenter.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    private val view: MyWeatherAppContract.MyWeatherAppView):
+    MyWeatherAppContract.MyWeatherAppPresenter {
+
+    override fun getWeatherForecast() {
+        view.showData()
+        model.getData()
+
+    }
+
+
```
Comment (BQuinteros) [inline app/src/main/java/com/example/myweatherapp/mvp/presenter/MyWeatherAppPresenter.kt]: pls remove extra lines

File: app/src/main/java/com/example/myweatherapp/mvp/model/MyWeatherAppModel.kt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.example.myweatherapp.mvp.model
+
+import com.example.myweatherapp.mvp.contract.MyWeatherAppContract
+
+class MyWeatherAppModel: MyWeatherAppContract.MyWeatherAppModel {
+
+    override fun getData() {
+        TODO("Not working yet")
```
Comment (BQuinteros) [inline app/src/main/java/com/example/myweatherapp/mvp/model/MyWeatherAppModel.kt]: when you run the app, this not crash here?

**Comment by Matias-Villalobo** [comment]
image

File: app/src/main/java/com/example/myweatherapp/mvp/model/MyWeatherAppModel.kt (hunk 2 above)
Comment (SartiMau) [reply app/src/main/java/com/example/myweatherapp/mvp/model/MyWeatherAppModel.kt]: yes, this crash.

**Review by SartiMau — COMMENTED** [review]
approved after comments

File: app/src/main/java/com/example/myweatherapp/mvp/presenter/MyWeatherAppPresenter.kt (hunk 1 above)
Comment (Matias-Villalobo) [reply app/src/main/java/com/example/myweatherapp/mvp/presenter/MyWeatherAppPresenter.kt]: Done thx

File: app/src/main/java/com/example/myweatherapp/mvp/model/MyWeatherAppModel.kt (hunk 2 above)
Comment (Matias-Villalobo) [reply app/src/main/java/com/example/myweatherapp/mvp/model/MyWeatherAppModel.kt]: Done thx guys

**Comment by Matias-Villalobo** [comment]
https://user-images.githubusercontent.com/73836200/106184491-600b6600-6180-11eb-8c85-4120181ee4fd.mp4

**Review by BQuinteros — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Comment by Matias-Villalobo** [comment]
Captura de pantalla 2021-01-28 155738

**Review by lucianoeli — APPROVED** [review]

**Pull request merged**